aea/crypto/ethereum.py

```diff
@@ -243,6 +255,27 @@ def send_transaction(
                 time.sleep(3.0)
         return tx_digest
 
+    def send_raw_transaction(self, tx_signed, tx_type) -> Optional[str]:
```

Comment:
This is almost exactly the same as above. Please refactor to avoid code duplication. For instance, the above method can call this method at line 244.